docs/machine-learning/how-to-guides/load-data-ml-net.md

```diff
@@ -51,7 +51,7 @@ Load columns as:
 - Individual columns like `Size` and `CurrentPrices` in the `HousingData` class.
 - Multiple columns at a time in the form of a vector like `HistoricalPrices` in the `HousingData` class.
 
-If you have a vector property, apply the [`VectorType`](xref:Microsoft.ML.Data.VectorTypeAttribute) attribute to the property in your data model. It's important to note that all of the elements in the vector need to be the same type.
+If you have a vector property, apply the [`VectorType`](xref:Microsoft.ML.Data.VectorTypeAttribute) attribute to the property in your data model. It's important to note that all of the elements in the vector need to be the same type. Keeping the columns separated allows for ease and flexibility of feature engineering, but for a very large number of columns, operating on the individual columns causes a performance impact.
```

Comment:
Apologies for the late comment.

The word "performance" has two connotations, either speed or predictive power in the model. I have to disambiguate rather often.

I'm uncertain how to best state this, though saying "`...causes a training speed impact.`" is likely suitable.

Side note: There's likely an impact to prediction throughput and latency, though I haven't tested this before.

Reply:
Thanks for the feedback. Clarifying language as part of #13691